### PR TITLE
bump assembly version 4.8 => 4.9

### PR DESCRIPTION
**Pull request by tomlm**
* Add missing contentFiles attribute
* remove over inclusion of json files in adaptive assembly
* add publishRepositoryUrl to msbuild

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.AI.LUIS/Microsoft.Bot.Builder.AI.Luis.csproj [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <PackageId>Microsoft.Bot.Builder.AI.Luis</PackageId>
     <Description>LUIS Middleware and Recognizers for the Microsoft Bot Builder SDK</Description>
     <Summary>This library implements C# classes for building bots using LUIS.</Summary>
+    <ContentTargetFolders>content</ContentTargetFolders>
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.AI.LUIS/Microsoft.Bot.Builder.AI.Luis.csproj:16]: Did you mean to add this?

File: libraries/Microsoft.Bot.Builder.AI.QnA/Microsoft.Bot.Builder.AI.QnA.csproj [hunk 2]
```diff
@@ -12,6 +12,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Classes for using QnA Maker Cognitive Service on the Microsoft Bot Builder SDK</Description>
     <Summary>This library implements C# classes for building bots using Microsoft Cognitive services.</Summary>
+    <ContentTargetFolders>content</ContentTargetFolders>
   </PropertyGroup>
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.AI.QnA/Microsoft.Bot.Builder.AI.QnA.csproj:16]: For .schema?

File: libraries/Microsoft.Bot.Builder.AI.LUIS/Microsoft.Bot.Builder.AI.Luis.csproj (hunk 1 above)
Comment (tomlm) [reply libraries/Microsoft.Bot.Builder.AI.LUIS/Microsoft.Bot.Builder.AI.Luis.csproj:16]: yes, that should have been there and wasn't

File: libraries/Microsoft.Bot.Builder.AI.QnA/Microsoft.Bot.Builder.AI.QnA.csproj (hunk 2 above)
Comment (tomlm) [reply libraries/Microsoft.Bot.Builder.AI.QnA/Microsoft.Bot.Builder.AI.QnA.csproj:16]: yeah, this causes content files to be included in package but not mapped into project folder

**Pull request merged**